Update docs for Workspace IA changes (#1360)
astronomer/docs: astro/manage-workspaces.md

### Update or Remove a Workspace User

> 1. Open a Workspace and click **Workspace Settings**.

Reviewer: ```suggestion
1. In the Cloud UI, select a Workspace.
2.  Click **Workspace Settings**.
```

2. Click the **Edit** button next to the user's entry in the **Member** table:

    ![Edit Workspace user button](/img/docs/edit-workspace-user.png)

> From here, you can either update a user's Workspace role or remove the user from the Workspace.

Reviewer: ```suggestion
3. Edit the user's name or role and then click **Update Member**.
```

## Update Workspace details

> 1. Open the Workspace in the Cloud UI

Reviewer: ```suggestion
1. In the Cloud UI, select a Workspace.
```

2. Click **Workspace Settings**, then click **Details**.

> 3. Click **Edit Details** and make changes. To save your changes, click **Update**.

Reviewer: ```suggestion
3. Click **Edit Details**, update the Workspace name or description, and then click **Update**.
```

### Delete a Workspace

> To delete a Workspace, click **Delete Workspace** in the **Workspace Settings** page. Note that you can delete a Workspace only if there are no active Deployments in the Workspace.

Reviewer 2: We'll be adding more tabs to Workspace Settings in the future. Maybe this would be helpful?
```suggestion
To delete a Workspace, click **Delete Workspace** in the overflow menu from any page within **Workspace Settings**. Note that you can delete a Workspace only if there are no active Deployments in the Workspace.
```

> To delete a Workspace, click **Delete Workspace** in the **Workspace Settings** page. Note that you can delete a Workspace only if there are no active Deployments in the Workspace.

Reviewer: ```suggestion
1.  In the Cloud UI, select a Workspace.
2. Click **Workspace Settings**.
3. Click the option menu at the top of the page and select **Delete Workspace**. This option isn't available when there are active Deployments in the Workspace.
```
